workflow/automation/org/nesi/srf_gen.sl

```diff
@@ -11,12 +11,12 @@ if [[ -n ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
     source $CUR_ENV/workflow/workflow/environments/helper_functions/activate_env.sh $CUR_ENV "mahuika"
 fi
 
-REL_CSV=${1:?REL_CSV argument missing}
+REL_FILEPATH=${1:?REL_CSV argument missing}
```

Comment:
Should this be REL_YAML? It is coincidentally consistent with https://github.com/ucgmsim/slurm_gm_workflow/blob/master/workflow/automation/org/kisti/srf_gen.pbs 
SLURM variables are positional whereas PBS cares about actual variable names. Also edit line 4 above
```
# must be run with sbatch srf_gen.sl [REL_CSV] [MGMT_DB_LOC] [REL_NAME]
```